designsafe/apps/api/views.py

```diff
@@ -155,3 +156,41 @@ def post(self, request):
             },
         )
         return HttpResponse("OK", status=202)
+    
+
+
+class SystemMonitorProxyApi(BaseApiView):
+    """"
+    Proxy API for fetching system queue data from Core Portal. 
+    """
+
+    def get(self, request, hostname):
+        CORE_PORTAL_API_BASE = "https://cep.tacc.utexas.edu/api/system-monitor"
```

Comment:
We'll want to use https://tap.tacc.utexas.edu/status/ as our source of truth for system status